etc/helm/pachyderm/templates/kube-event-tail/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.kubeEventTail.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kube-event-tail
+    suite: pachyderm
+  name: kube-event-tail
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kube-event-tail
+      suite: pachyderm
+  template:
+    metadata:
+      labels:
+        app: kube-event-tail
+        suite: pachyderm
+    spec:
+      containers:
+        - name: kube-event-tail
+          env:
+            - name: DEBUG_ADDRESS
+              value: 0.0.0.0:8081
+            - name: NAMESPACE
+{{- if eq .Values.kubeEventTail.clusterScope true }}
+              value: ""
+{{ else }}
+              value: {{ .Release.Namespace }}
+{{- end }}
+          image: docker.io/jrockway/kube-event-tail:v0.0.6
+          imagePullPolicy: IfNotPresent
```

Comment:
We also have to let customers pass in an image pull secret.  For reasons.  Check out the other manifests for an example.